### PR TITLE
Get GitHub access token from build api

### PR DESCRIPTION
**Pull request by irushx**

### CODE REVIEW

**Review by 928PJY — APPROVED** [review]
:shipit:

**Review by herohua — CHANGES_REQUESTED** [review]
temporarily holding the approval

**Review by herohua — APPROVED** [review]
You can start testing on PubDev now.

File: docs/specs/git.yml [hunk 1]
```diff
@@ -459,8 +459,6 @@ repos:
           Osmond creates this
 inputs:
   docfx.yml: |
-    secrets:
```
Comment (928PJY) [inline docs/specs/git.yml:462]: We should cover the two cases in spec test:
- token from secrets
- token from API call.

**Review by 928PJY — APPROVED** [review]

**Pull request merged**